src/load/load.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import logging
+import os
+
+import load.context
+
+from load.load import load_secrets
+from lib.logging import setup_logging
+from lib.validations import (
+  hostname_agrees_with_artsy_environment
+)
+
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter
+  )
+  parser.add_argument(
+    'artsy_env',
```

Comment:
Any reason why these args need `artsy_` prefixes? Seems like a generic env and project arg. We use such args elsewhere (e.g. in zenith) without prefixing.

Reply:
right on. will remove artsy prefix.

Reply:
done